pages/_app.js

```diff
@@ -1,7 +1,12 @@
 import { useEffect } from 'react'
 import PropTypes from 'prop-types'
-import { ThemeProvider } from '@material-ui/core/styles'
-import CssBaseline from '@material-ui/core/CssBaseline'
+// NOTE: If I change the import below to go to @mui
```

Comment:
Maybe this is a help for this (it would seem to be an issue mentioned here)
https://mui.com/material-ui/migration/v5-style-changes/

Reply:
Yeah, I came across that and it worked. I'll remove the comments since they don't apply now.